src/utilities/theme/types.ts

```diff
@@ -29,6 +29,8 @@ export interface ThemeConfig {
     warning?: string;
     highlight?: string;
     success?: string;
+    peach?: string;
+    cranberry?: string;
```

Comment:
Peach and cranberry are what these are called in figma. We can probably think of better names for these

Reply:
Agreed, we had some early discussion around these as decorative colors. The word "decorative" is so vague, though. Any thoughts?

Reply:
extended, complementary? Could they be bundled under 1 role?